gluten-arrow/src/main/scala/org/apache/spark/sql/utils/SparkArrowUtil.scala

```diff
@@ -97,7 +98,7 @@ object SparkArrowUtil {
             .map(
               field =>
                 toArrowField(
-                  ConverterUtils.normalizeStructFieldName(field.name),
+                  normalizeColName(field.name),
```

Comment:
@zhztheplayer, looks `normalizeColName` is not equivalent to `normalizeStructFieldName` which makes the conversion disabled for CH backend.

Reply:
It seems gluten-arrow only used for velox backend.

Reply:
I see. Thanks!

Reply:
Yes. Let's keep `gluten-arrow` an individual module so future refactors can be made easier. Minor code duplications like this one will be acceptable.